data/101/912/923/101912923.geojson

```diff
@@ -517,10 +518,10 @@
824,36.86278],[30.72609,36.863652],[30.725004,36.863896],[30.72527,36.864767],[30.724185,36.865011],[30.72445,36.865882],[30.72011,36.866856],[30.720375,36.867728],[30.718205,36.868214],[30.71847,36.869086],[30.717385,36.86933],[30.717916,36.871073],[30.716831,36.871316],[30.717096,36.872188],[30.716011,36.872431],[30.716277,36.873303],[30.706508,36.875493],[30.706774,36.876364],[30.705688,36.876608],[30.705954,36.877479],[30.704868,36.877723],[30.705134,36.878594],[30.704048,36.878837],[30.704314,36.879709],[30.703228,36.879952],[30.703759,36.881696],[30.702674,36.881939],[30.702939,36.882811],[30.701854,36.883054],[30.702119,36.883926],[30.697777,36.884898],[30.697512,36.884027],[30.69534,36.884513],[30.695075,36.883641],[30.691818,36.88437],[30.691553,36.883499],[30.689382,36.883985],[30.689117,36.883113],[30.685529,36.883916]],[[30.733112,36.913294],[30.729854,36.914025],[30.729588,36.913153],[30.728502,36.913397],[30.728236,36.912525],[30.724979,36.913256],[30.724713,36.912384],[30.719283,36.913601],[30.719017,36.91273],[30.717932,36.912973],[30.7174,36.91123],[30.716314,36.911473],[30.715251,36.907987],[30.716337,36.907744],[30.716071,36.906872],[30.722586,36.905411],[30.72232,36.90454],[30.723405,36.904296],[30.72314,36.903425],[30.726397,36.902694],[30.726663,36.903566],[30.727748,36.903322],[30.72828,36.905066],[30.729366,36.904822],[30.729632,36.905694],[30.730718,36.90545],[30.732048,36.909808],[30.733134,36.909564],[30.733932,36.912179],[30.732846,36.912423],[30.733112,36.913294]],[[30.725559,36.953167],[30.725293,36.952295],[30.724206,36.952539],[30.723142,36.949053],[30.724228,36.948809],[30.723962,36.947938],[30.727221,36.947207],[30.726955,36.946336],[30.728041,36.946092],[30.727775,36.945221],[30.728861,36.944977],[30.728595,36.944106],[30.729681,36.943862],[30.728617,36.940376],[30.729703,36.940132],[30.728904,36.937518],[30.727818,36.937761],[30.727552,36.93689],[30.726466,36.937133],[30.726199,36.936262],[30.725113,36.936505],[30.724049,36.933019],[30.722963,36.933263],[30.722165,36.930648],[30.723251,36.930405],[30.722985,36.929533],[30.725157,36.929046],[30.724891,36.928175],[30.732493,36.92647],[30.73276,36.927341],[30.733846,36.927098],[30.734644,36.929712],[30.731386,36.930443],[30.732184,36.933058],[30.733271,36.932814],[30.734602,36.937171],[30.733515,36.937415],[30.734314,36.94003],[30.739745,36.938811],[30.740012,36.939683],[30.74327,36.938952],[30.744602,36.943309],[30.743516,36.943553],[30.743782,36.944424],[30.744869,36.94418],[30.745135,36.945052],[30.746222,36.944808],[30.745955,36.943937],[30.747041,36.943693],[30.746775,36.942822],[30.75112,36.941846],[30.751919,36.944461],[30.750833,36.944704],[30.7511,36.945576],[30.750013,36.94582],[30.75028,36.946691],[30.740503,36.948885],[30.740236,36.948013],[30.73915,36.948257],[30.738884,36.947386],[30.737797,36.947629],[30.736998,36.945015],[30.732653,36.945989],[30.733185,36.947732],[30.734272,36.947489],[30.735337,36.950975],[30.725559,36.953167]]],"type":"Polygon"}
```

Comment:
Should this geometry get an update? It looks to be the old Quaatroshapes geom still, not including the airport or the left side of the city (I don't know what it should really look like).

Reply:
I've updated this locality with a more accurate polygon (which includes the airport) in 3c51e65.